symphony-bdk-core/src/test/java/com/symphony/bdk/core/config/BdkConfigParserTest.java

```diff
@@ -11,13 +11,34 @@
 import org.junit.jupiter.api.Test;
 
 import java.io.InputStream;
+import java.lang.reflect.Field;
+import java.util.Map;
 
 public class BdkConfigParserTest {
 
+    //this is a disgusting hack that should only be allowed in unit tests
+    void hackEnvVar(String key, String value) throws NoSuchFieldException, IllegalAccessException {
+        Map<String, String> env = System.getenv();
+        Field field = env.getClass().getDeclaredField("m");
+        field.setAccessible(true);
+        if(value==null) {
+          ((Map<String, String>) field.get(env)).remove(key);
+        }
+        else {
+          ((Map<String, String>) field.get(env)).put(key, value);
+        }
```

Comment:
Could you reformat the file, our [`.editorconfig`](https://github.com/finos/symphony-bdk-java/blob/main/.editorconfig) should give: 
```java
    Map<String, String> env = System.getenv();
    Field field = env.getClass().getDeclaredField("m");
    field.setAccessible(true);
    if (value == null) {
      ((Map<String, String>) field.get(env)).remove(key);
    } else {
      ((Map<String, String>) field.get(env)).put(key, value);
    }
```